### PR TITLE
Add jets for min/max.

### LINKED ISSUE #456: min/max are unjetted
NBD but @belisarius222 and I noticed this during our dev talk today.

### PR DESCRIPTION
**Pull request by sigilante**
Resolves #456.

Written live for a tutorial with @belisarius222

### CODE REVIEW

File: pkg/noun/jets/a/max.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/// @file
+
+#define MAX(i, j) (((i) > (j)) ? (i) : (j))
```
Comment (joemfb) [inline pkg/noun/jets/a/max.c:3]: We already have this as `c3_max()`, lets use that.

File: pkg/noun/jets/a/max.c [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      return u3k((a_w > b_w) ? a : b);
+    }
+    else {
+      mpz_t   a_mp, b_mp;
+      u3_noun max;
+
+      u3r_mp(a_mp, a);
+      u3r_mp(b_mp, b);
+
+      max = (mpz_cmp(a_mp, b_mp) >= 0) ? a : b;
```
Comment (joemfb) [inline pkg/noun/jets/a/max.c:31]: This should be `> 0` for consistency.

File: pkg/noun/jets/a/min.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/// @file
+
+#define MIN(i, j) (((i) < (j)) ? (i) : (j))
```
Comment (joemfb) [inline pkg/noun/jets/a/min.c:3]: `c3_min()`

File: pkg/noun/jets/a/min.c [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      return u3k((a_w < b_w) ? a : b);
+    }
+    else {
+      mpz_t   a_mp, b_mp;
+      u3_noun min;
+
+      u3r_mp(a_mp, a);
+      u3r_mp(b_mp, b);
+
+      min = (mpz_cmp(a_mp, b_mp) <= 0) ? a : b;
```
Comment (joemfb) [inline pkg/noun/jets/a/min.c:31]: `< 0`

**Review by joemfb — CHANGES_REQUESTED** [review]
These look good. Kudos for that "cell or zero" crash checks, those are subtle.

A couple nits:

**Review by joemfb — APPROVED** [review]
this will do

**Comment by ashelkovnykov** [comment]
> Kudos for that "cell or zero" crash checks, those are subtle.

I don't understand - why are the `bail` checks "cell or zero"?

**Pull request merged**